crates/bevy_ecs/src/schedule/state.rs

```diff
@@ -279,7 +279,11 @@ impl<T: Component + Clone + Eq> State<T> {
     }
 
     pub fn inactives(&self) -> &[T] {
-        &self.stack[0..self.stack.len() - 2]
+        if let Some(idx) = self.stack.len().checked_sub(2) {
+            &self.stack[0..idx]
+        } else {
+            &[]
+        }
```

Comment:
```suggestion
        self.stack.split_last().map(|(_last, rest)| rest).unwrap_or(&[])
```

Reply:
A plain unwrap would be preferred IMO, since the stack must always have at least one element.